configuration/openmrs/i18n/registration/locale_fr.json

```diff
@@ -0,0 +1,131 @@
+{
+  "_comment": "This is a concatenation of the Bahmni Config and Bahmni Apps localization files. Refer to this thread https://talk.openmrs.org/t/bahmni-changes-in-locale-en-json-should-i-update-transifex-as-well/14283?u=mksrom to better understand why.",
+  "ACCEPT": "Accepter",
+  "ATTRIBUTE_SEARCH": "Attributs",
+  "CHOOSE_ANSWER_FROM_DROPDOWN_LABEL": "Selectionner une réponse",
+  "CLICK_PATIENT_PHOTO_LABEL": "Prendre photo",
+  "CONCEPT_NOT_FOUND_MESSAGE": "Impossible de trouver le Concept",
+  "CONFIRM_PATIENT_PHOTO_LABEL": "Valider",
+  "DELETE_LABEL": "Delete",
+  "CUSTOM_SEARCH": "Entrer",
+  "DISPLAY_CONTROL_ERROR_MESSAGE_KEY": "Pas {{ displayControl }} pour ce patient.",
+  "EDIT_LABEL": "Modifier",
+  "FEE_INFORMATION_LOCALE_KEY": "Frais de consultation",
```

Comment:
I think we should rather change that in the parent Haiti config, what do you think?

Reply:
That would be great actually. Cause I had to duplicate that registration directory in i18n folder into easyDoc to override what is in the parent.